### PR TITLE
stage 1 java exceptions theory

### PR DESCRIPTION
**Pull request by GretchenkoAS**

### CODE REVIEW

File: stage #1/module #1. Exceptions/content.md [hunk 1]
```diff
@@ -1,20 +1,175 @@
...
 
-### Intro
+## What is an Exception?
+
+### Throwing an exception
+**An exception** is an event that occurs during the execution of a program that disrupts the normal flow of instructions.
+
+**Throwing an exception** is the process of creating an exception object and handing it to the runtime system.
+
+![image](./media/image1.png)
```
Comment (egoist301) [inline stage #1/module #1. Exceptions/content.md:20]: Could you use names that will reflect the information of the file?

File: stage #1/module #1. Exceptions/content.md [hunk 2]
```diff
@@ -1,20 +1,175 @@
-# How to write your own exception classes?
+# Exceptions
```
Comment (egoist301) [inline stage #1/module #1. Exceptions/content.md:1]: Maybe it's worth splitting this module into several lessons?

File: stage #1/module #1. Exceptions/content.md (hunk 1 above)
Comment (GretchenkoAS) [reply stage #1/module #1. Exceptions/content.md:20]: Sure, thanks

File: stage #1/module #1. Exceptions/content.md (hunk 2 above)
Comment (GretchenkoAS) [reply stage #1/module #1. Exceptions/content.md:1]: I don't know, what do others think?

File: stage #1/module #1. Exceptions/lesson #1. Overview/owerview.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Exceptions Overview
+
+## Agenda
```
Comment (egoist301) [inline stage #1/module #1. Exceptions/lesson #1. Overview/owerview.md:3]: Could you use the link for your points from agenda?
[Materials](#materials)
## Materials

Comment (GretchenkoAS) [reply stage #1/module #1. Exceptions/lesson #1. Overview/owerview.md:3]: yes, thanks

File: stage #1/module #1. Exceptions/lesson #1. Overview/owerview.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
++ [What is an Exception?](#what-is-an-exception)
++ [The Three Kinds of Exceptions](#the-three-kinds-of-exceptions)
++ [Materials](#materials)
+
+## <a id="what-is-an-exception"></a> What is an Exception?
```
Comment (egoist301) [inline stage #1/module #1. Exceptions/lesson #1. Overview/owerview.md:8]: Could you wrap header with tag `<a>`? For example
`<a id="what-is-an-exception">What is an Exception?</a>`

Comment (GretchenkoAS) [reply stage #1/module #1. Exceptions/lesson #1. Overview/owerview.md:8]: yes, thanks

**Pull request closed without merging**